setup.py

```diff
@@ -64,22 +61,27 @@
     ],
     "gpu": [
         "nvidia-ml-py3>=7.352.0",
-        "tensorflow-gpu==1.15.4",  # compiled with cuda 10.0
+        "tensorflow-gpu>=1.15.0,<2",  # compiled with CUDA 10.0
+        "torch==1.2.0",  # last os-common version with CUDA 10.0 support
         "fastai>=1.0.46,<2",
-        "torch==1.2.0",  # last os-common version with cuda 10.0 support
     ],
     "spark": [
         "databricks_cli>=0.8.6,<1",
         "pyarrow>=0.8.0,<1.0.0",
         "pyspark>=2.4.5,<3.0.0",
     ],
+    "xlearn": [
```

Comment:
@anargyri can you please confirm, xlearn goes here or does it go inside extras_requre?